ThirdPartyAdapters/vungle/vungle/build.gradle

```diff
@@ -35,12 +36,13 @@ android {
 }
 
 dependencies {
-    implementation ('com.vungle:publisher-sdk-android:6.7.0') {
+    implementation (name: 'vungle-android-sdk-6.7.0-dax', ext:'aar') {
         transitive=true
     }
+    implementation 'com.google.code.gson:gson:2.8.6'
```

Comment:
do we need the gson and okhttp deps here in adapter, shouldnt they come from the sdk .aar

Reply:
aar doesn't provide dependency by itself, and publisher might want to use another version and will have conflicts if we add them as jar to vungle sdk aar

Reply:
@alexander-gubov ur right, I got confused with location of pom , they are outside .aar and only generated when we deploy SDK as maven artifact on jitpack/bintray